### DISCUSSION TITLE
Questions about the proportion of metabolic contribution of different enzymes

### DISCUSSION THREAD

**Discussion by SCZ1174** [Open-Systems-Pharmacology/Forum#1476] [Questions & Problems] [answered]
Hello everyone.
If multiple metabolizing enzymes are involved in the metabolism of a drug, but the enzymatic kinetic information is incomplete and only the metabolic ratios of the various enzymes are known. I want to optimize it directly by Monte Carlo simulation, how do I fix the metabolic ratio of each enzyme to optimize the parameters?
If there is a prototype drug excreted directly from the kidney, can I fix the ratio of hepatic to renal clearance separately?
An example is shown in the following figure: 
![image](https://user-images.githubusercontent.com/86552551/224670787-a330147f-2af7-48bb-b57d-5cf1298d8ea0.png)
(Yeung CHT, Beers JL, Jackson KD, Edginton AN. Verifying in vitro-determined enzyme contributions to cannabidiol clearance for exposure predictions in human through physiologically-based pharmacokinetic modeling. CPT Pharmacometrics Syst Pharmacol. 2022 Dec 20. doi: [10.1002/psp4.12908](https://doi.org/10.1002/psp4.12908). Epub ahead of print. PMID: 36540909.)

**Comment by Yuri05** [comment, marked as answer]
For each enzyme: create and Import an observed data set with just one point: `{simulation_end_time; fraction metabolized}`
Then you can optimize the corresponding Total fraction metabolized of the enzyme vs. this data point

**Reply by SCZ1174** [reply]
Thanks a lot! I will try.

**Comment by ahmedsanot2009** [comment]
@Yuri05 

This is an interesting point, Could you please elaborate more on this issue?

**Reply by SCZ1174** [reply]
Supplement: Simple process (for reference):
- Predicted data: Simulations→Define Settings and Run→Kindey/Organism→Fraction excreted to urine/Total fraction of dose-.
- Observed data: X-axis="Time", Y-axis="Fraction of original form excreted to urine/Total fraction of dose-contributed by metabolizing enzymes".
- Both PK and fraction were fitted in the "Parameter Identifications" module.
The final results are referenced below:
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/86552551/8f298c50-f651-4547-a75a-4dfb742c6301)
